dockers/docker-snmp/snmp_yml_to_configdb.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/lib/env python
```

Comment:
Since we are migrating from Python 2 to Python 3, both are currently installed in the system, so you should change the shebang to `#!/usr/lib/env python3` to ensure it is run using the Python 3 interpreter.

Also, please ensure this file is executable. Unfortunately, GitHub does not show the permissions on newly-added files (they really need to add this feature).

Reply:
Updated to Python3.  Ran command to update new file to 755 but not sure it this is going through or not.